static/src/js/reducers/toastNotifications.js

```diff
@@ -0,0 +1,54 @@
+import {
+  PUSHING_NOTIFICATION,
+  NOTIFICATION_PUSHED,
+  NOTIFICATION_DIMISSED
+} from '../constants/actionTypes'
+
+const initialState = {
+  history: new Map()
```

Comment:
I'm unsure of what this is for and I'm unable to see anything being written to Redux as a result of these -- my assumption is to try and keep a history of what actions have been taken but that use case doesn't exist for us at the moment.

Reply:
You are correct, it was merely to keep a history of notifications. The intent was to at some point allow users to recall notifications that may have been accidentally dismissed. For some reason, Redux dev tools cannot parse the Map and just shows an empty object. However, my tests confirm that history is being stored. If this is not needed, I will remove it. Can you please confirm?

Reply:
Yea, let's remove it for now, I don't see a need for it in the foreseeable future.